.github/ISSUE_TEMPLATE/end_game.md

```diff
@@ -61,6 +61,7 @@ assignees: ''
   - [ ] [Clickhouse Data Connector](https://github.com/spiceai/quickstarts/blob/trunk/clickhouse/README.md)
   - [ ] [Snowflake Data Connector](https://github.com/spiceai/quickstarts/blob/trunk/snowflake/README.md)
   - [ ] [GraphQL Data Connector](https://github.com/spiceai/quickstarts/blob/trunk/graphql/README.md)
+  - [ ] [GitHub Data Connector](https://github.com/spiceai/quickstarts/blob/trunk/github/README.md)
```

Comment:
Also, File, FTP, HTTP data connectors?

Reply:
Ftp connector is already here. And we don't have file and http quickstarts yet.